frontend/src/metabase-types/api/writeback-form-settings.ts

```diff
@@ -1,6 +1,6 @@
 import type { ParameterId } from "./parameters";
 
-export type FormType = "inline" | "modal";
+export type ActionDisplayType = "form" | "button";
```

Comment:
the way this feature is going, it seems like `form` and `button` are the better values here rather than `inline` and `modal`, it seems like it makes sense to change this sooner rather than later and be left with some puzzling values in the code that we'll have to mentally translate.